css/index.css

```diff
@@ -1 +1,306 @@
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+	font-family: 'Titillium Web', sans-serif;
+	font-size: 62.5%; 
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* -------------------------------------- */
+
+.navigation {
+
+	display: flex;
+	flex-direction: row;
+	justify-content: center;
+	max-width: 75%;
+	
+}
+
+a {
+	margin: 3%;
+	font-weight: bold;
+	font-size: 1.8rem;
+	padding-top: 24px;
+	padding-bottom: 16px;
+	padding-right: 32px;
+	color: #E2E5E8;
+	text-decoration: none;
+	max-width: 75%;
+}
+
+
+.idea-img {
+	display: inline-block;
+	position: absolute;
+	top: 45px;
+	right: 175px;
+	
+}
+
+.header {
+    display: flex;
+    justify-content: center;
+    align-content: space-around;
+    margin: 2rem;
+}
+
+.header h1 {
+	font-size: 10rem;
+	text-align: center;
+}
+
+.getstarted-button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    align-content: flex-start;
+	padding-right: 500px;
+	font-weight: bold;
+	font-size: 1.5rem;
+	
+}
+
+.top-section-header {
+    display: flex;
+    justify-content: space-around;
+    align-content: column;
+    margin: 4rem;
+    padding-top: 45px;
+	border-top: 2px solid black;
+	font-size: 2rem;
+}
+
+.top-section {
+    display: flex;
+    justify-content: center;
+    align-content: column;
+    /* margin: 24px; */
+	margin-right: 12px;
+	font-size: 1.8rem;
+	line-height: 1.25;
+}
+
+.middle-img {
+    display: flex;
+    justify-content: space-around;
+    align-content: column;
+	margin: 0 auto;
+	padding: 2% 0%;
+}
+
+
+.bottom-section-header {
+    display: flex;
+    justify-content: space-evenly;
+    align-content: column;
+    margin: 4rem;
+	margin-right: 12px;
+	font-size: 2rem;
+}
+
+.bottom-section {
+    display: flex;
+    justify-content: center;
+    align-content: column;
+    margin: 4rem;
+    border-bottom: 2px solid black;
+	padding-bottom: 16px;
+	font-size: 1.8rem;
+	line-height: 1.25;
+   
+}
+
+
+.contact-header {
+    display: flex;
+    justify-content: center;
+    align-content: flex-start;
+    padding-top: 12px;
+    margin: 4rem;
+	padding-right: 750px;
+	font-size: 1.8rem;
+	line-height: 1.25;
+}
+
+.contact {
+    display: flex;
+    justify-content: center;
+    align-content: flex-start;
+    padding-top: 16px;
+    margin: 24px;
+	padding-right: 675px;
+	font-size: 1.8rem;
+	line-height: 1.25;
+}
+
+.footer {
+	display: flex;
```

Comment:
It's a technical necessity and in this case it's a small difference, but I encourage you to try to use consistent spacing to make your code easier for you and others to read.